public/docs/ts/latest/guide/browser-support.jade

```diff
@@ -0,0 +1,190 @@
+  
+.alert.is-important 
+  span The polyfills described here under are the ones which, others may be required for your application.
+  span  On top of that, they will not magically transform an old and slow browser into a modern and fast one.
+  
+:marked
+  ## Mandatory polyfills ##
+  For each supported browser, these are the polyfills which are required to run an application:
+  
+table
+  tr
+    th Browsers (desktop & mobile)
+    th Polyfills required
+  tr
+    td Chrome, Firefox, Edge, Safari 9+
+    td None
+  tr
+    td Safari 7 & 8, IE10 & 11, Android 4.1+
+    td ES6
+  tr
+    td IE9
+    td ES6<br>classList
+    
+    
+:marked
+  ## Optional polyfills ##
+  On top of the mandatory polyfills, some features of Angular may require some extra polyfills.
+  
+  For example, the animations use the standard web animation API, which is only available in Chrome and Firefox today.
+  In order to use this feature in other browsers, the correct polyfill is needed.
+  
+  Here are the features which may require additional polyfills:
+
+table
+  tr
+    th Feature
+    th Polyfill
+    th Browsers (desktop & mobile)
+  tr
+    td
+      a(href="./animations.html") Animations
+    td Web Animations
+    td All but Chrome and Firefox<br>Not supported in IE9
+  tr
+    td 
+      a(href="../api/common/index/DatePipe-class.html") Date
+      span ,  
+      a(href="../api/common/index/CurrencyPipe-class.html") currency
+      span , 
+      a(href="../api/common/index/DecimalPipe-class.html") decimal
+      span  and 
+      a(href="../api/common/index/PercentPipe-class.html") percent
+      span  pipes
+    td Intl API
+    td All but Chrome, Firefox, Edge and IE11
+  tr
+    td 
+      a(href="../api/common/index/NgClass-directive.html") NgClass
+      span  on SVG elements
+    td classList
+    td IE10, IE11
+  tr
+    td
+      a(href="./server-communication.html") Http
+      span  when sending and receiving binary data
+    td Typed Arrays<br>Blob<br>FormData
+    td IE 9
+
+:marked
+  ## Suggested polyfills ##
+  Below are the polyfills which are used to test the framework itself. They are a good starting point for an application. 
+
+table
+  tr
+    th Polyfill
+    th Library
+    th Licence
+    th Size*
+  tr
+    td ES6
+    td 
+      a(href="https://github.com/zloirock/core-js") https://github.com/zloirock/core-js
+    td MIT
+    td 27.4KB
+  tr
+    td classList
+    td
+      a(href="https://github.com/eligrey/classList.js") https://github.com/eligrey/classList.js
+    td Public domain
+    td 1KB
+  tr
+    td Intl
+    td
+      a(href="https://github.com/andyearnshaw/Intl.js ") https://github.com/andyearnshaw/Intl.js 
+    td MIT / Unicode licence
+    td 13.5KB
+  tr
+    td Web Animations
+    td
+      a(href="https://github.com/web-animations/web-animations-js") https://github.com/web-animations/web-animations-js
+    td Apache
+    td 14.8KB
+  tr
+    td Typed Arrays
```

Comment:
the same for Blob and FormData - they are not really needed unless you are using particular browser features in your app at which point you are responsible for figuring out what polyfills you need. none of these should be used directly by angular's code unless I'm missing something.